tests/migration/migration.go

```diff
@@ -853,7 +853,10 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 
 				dataVolume := libdv.NewDataVolume(
 					libdv.WithRegistryURLSourceAndPullMethod(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), cdiv1.RegistryPullNode),
-					libdv.WithStorage(libdv.StorageWithStorageClass(sc)),
+					libdv.WithStorage(
+						libdv.StorageWithStorageClass(sc),
+						libdv.StorageWithAccessMode(k8sv1.ReadWriteOnce),
```

Comment:
Ah, a side effect of https://github.com/kubevirt/kubevirt/pull/12072. Sorry, I promise it was for a good cause

Reply:
:+1: thanks for the context, maybe we should retrospectively review if this might be a problem in other places

Reply:
I took a look at the virt tests just now. This seems like the only issue.
We've had issues come up like this for storage tests as well, hopefully I've identified all of them by now..